.github/workflows/ci-windows.yml

```diff
@@ -55,6 +55,16 @@ jobs:
       env:
         CC: clang-cl
         CXX: clang-cl
+    - name: Build SVT
+      if: steps.setup.outputs.ext-cache-hit != 'true'
+      working-directory: ./ext
+      run: ./svt.cmd
+      # Visual Studio 2022 has an issue starting at 17.8.0 which might cause
+      # AVX-512 instructions to be emitted in non-AVX-512 code. See
+      # https://github.com/AOMediaCodec/libavif/issues/2033#issuecomment-1960062751.
```

Comment:
The SEH exception code is different: 0xc0000005 vs. 0xc000001d in https://github.com/AOMediaCodec/libavif/issues/2033. So we should not assume this is the same Visual Studio 2022 bug.

I haven't found a webpage that tabulates the SEH exception codes, but my web searches shows that 0xc0000005 is access violation and 0xc000001d is illegal instruction.